CodeAssignment_MVVM_Combine/Sources/Controllers/DetailViewController.swift

```diff
@@ -0,0 +1,309 @@
+            }
+        } else {
+            userImageView.image = defaultImage
+        }
+    }
+    /** 数字をdecimal StyleのString型としてformatする
+    - Repositoryの中に、funcやcomputed propertyとして定義するより、ここで関数として作った理由は、format作業をする対象が複数だからである
+     */
+    private func formatNumberToStringWithSeparator(_ number: Int) -> String {
+        let numberFormatter = NumberFormatter()
+        numberFormatter.numberStyle = .decimal
+        return numberFormatter.string(from: NSNumber(value: number)) ?? ""
+    }
+    
+    private func setupUI() {
+        view.backgroundColor = .secondarySystemBackground
+        
+        setupNavigationController()
+        setAddSubViews()
+        setupConstraints()
+    }
+    
+    private func setupNavigationController() {
+        let appearance = UINavigationBarAppearance()
+        appearance.configureWithOpaqueBackground()
+        // HomeVCに戻ると、navigationBarの背景が変わってしまうので、ここもbackgroundColorを設定
+        appearance.backgroundColor = .secondarySystemBackground
+        // NavigationBarの下部線を隠す
+        appearance.shadowColor = .clear
+        navigationController?.navigationBar.standardAppearance = appearance
+    }
+    
+    private func makeCountLabel() -> UILabel {
+        let label = UILabel()
+        label.font = .systemFont(ofSize: 18, weight: .regular)
+        label.textColor = .black.withAlphaComponent(0.7)
+        return label
+    }
+    
+    private func setAddSubViews() {
+        userImageContentView.addSubview(userImageView)
+        backgroundCardView.addSubview(mainStackView)
+        scrollView.addSubview(backgroundCardView)
+        view.addSubview(scrollView)
+    }
+    
+    private func setupConstraints() {
+        scrollView.snp.makeConstraints { constraint in
+            // ScrollIndicatorの挙動がBottomのSafeAreaを超えてしまうので、equalToSuperViewに変更
+            constraint.edges.equalToSuperview()
+        }
+        
+        backgroundCardView.snp.makeConstraints { constraint in
+            constraint.top.equalTo(scrollView.contentLayoutGuide.snp.top).offset(30)
+            constraint.width.equalTo(scrollView.snp.width).offset(-(Const.leftPadding + Const.rightPadding))
+            constraint.centerX.equalTo(scrollView.snp.centerX)
+            constraint.bottom.equalTo(scrollView.contentLayoutGuide.snp.bottom).offset(-30)
+        }
+        
+        userImageView.snp.makeConstraints { constraint in
+            constraint.top.equalTo(userImageContentView.snp.top)
+            constraint.height.equalTo(250)
+            constraint.width.equalTo(250)
+            constraint.centerX.equalTo(userImageContentView.snp.centerX)
+        }
+        
+        userNameLabel.snp.makeConstraints { constraint in
+            constraint.top.equalTo(userImageView.snp.bottom).offset(10)
+            constraint.leading.equalTo(backgroundCardView.snp.leading).offset(Const.leftPadding)
+            constraint.trailing.equalTo(backgroundCardView.snp.trailing).offset(-Const.rightPadding)
+        }
```

Comment:
userNameLabelはもうStackViewの中に入ってるから、offsetとかを指定する必要はないね。leading/trailingもmainStackViewに対してしているはずだから、ここはこんな感じになるはず。
```suggestion
        userImageView.snp.makeConstraints { constraint in
            constraint.top.equalTo(userImageContentView.snp.top)
            constraint.height.equalTo(250)
            constraint.width.equalTo(250)
            constraint.centerX.equalTo(userImageContentView.snp.centerX)
            constraint.bottom.equalTo(userImageContentView.snp.bottom)
        }
```

Reply:
[50e3a16]にて、コード修正しました！ご確認お願いします！